### PR TITLE
docs: Add configuration for SEO

### PR DESCRIPTION
**Pull request by mikemckiernan**
* Set the canonical URL for pages to the main branch.

* Add a Google tag for Google Analytics.

### CODE REVIEW

**Comment by mikemckiernan** [comment]
rerun tests

**Comment by mikemckiernan** [comment]
rerun tests

**Comment by mikemckiernan** [comment]
rerun tests

**Comment by mikemckiernan** [comment]
rerun tests

**Review by jperez999 — APPROVED** [review]

File: docs/source/conf.py [hunk 1]
```diff
@@ -71,6 +71,7 @@
 html_theme = "sphinx_rtd_theme"
 html_theme_options = {
     "navigation_depth": 3,
+    "analytics_id": "G-NVJ1Y1YJHK",
```
Comment (EvenOldridge) [inline docs/source/conf.py:74]: I'm assuming this id is public and not private?

**Review by EvenOldridge — COMMENTED** [review]
lgtm

**Pull request merged**